### ISSUE TITLE
JSON output and conditional SDL compilation

### ISSUE THREAD

**Issue by nicopace** [simonwunderlich/FFT_eval#11]
Hi Guys,

I'm working on a Spectrum Analyzer module for LuCI, compatible with OpenWRT/LEDE.
For this I need to separate the SDL functionality and the data conversion functionality, and make some sort of JSON output.

I've been trying to do it myself based on @altergui work here: https://github.com/libremesh/FFT_eval/commit/578707430e09eb7a8e3bc0a7ca1e0067e30ed0d9 but it is taking a lot of energy from me.

I wondered if you could be able to do this. That would make my part easier and I can focus on creating the LuCI modules and the ubus daemons for this.

Basically, the proposal is to add a mode on the code that allows to export the data information as JSON, and a conditional build switch to split the SDL dependencies.

Thanks!

**Comment by nicopace** [comment]
I already sent a PR for the first part, the SDL conditional compilation: https://github.com/simonwunderlich/FFT_eval/pull/12
